src/java/io/bazel/rulesscala/scalac/ScalacWorker.java

```diff
@@ -312,4 +315,30 @@ private static void copyResourceJars(String[] resourceJars, Path dest) throws IO
       extractJar(jarPath, dest.toString(), null);
     }
   }
+
+  private static <T> String[] appendToString(String[] init, List<T> rest) {
```

Comment:
These methods were originally in the shared `GenericWorker`, but were only every used by the Scalac worker.

Reply:
You moved them as is, right? No changes whatsoever?

Reply:
Yep, they were copied over without any changes.